src/pages/schedule.tsx

```diff
@@ -31,7 +31,7 @@ export default function Schedule({
 }: Awaited<ReturnType<typeof getStaticProps>>["props"]) {
   return (
     <Layout
-      title="Transcripts"
+      title="Q&amp;A Schedules"
```

Comment:
Let's use the same pluralization as the navbar link and page title:
```suggestion
      title="Q&amp;A Schedule"
```